pkg/cli/codeset/register.go

```diff
@@ -102,11 +102,33 @@ func (o *RegisterOptions) run() error {
 	}
 
 	fmt.Printf("Codeset %s successfully registered\n", *codeset.URL)
+
 	if result.Username != nil {
-		fmt.Printf("Username for accessing the project %s\n", *result.Username)
+		if viper.GetString("Username") != *result.Username {
+			fmt.Println("Saving new username into config file as current username.")
```

Comment:
Something to think about for next time: we only have one slot for username/password and registering another codeset will overwrite the user/pass used for the previously registered codeset. Clients like kubectl and docker deal with this by keeping a full list of contexts in the configuration file. They even warn you that the passwords are kept in clear and that might be a security problem.

Reply:
Yes, that's true, I've thought about that.

We could e.g. have a list of users grouped under project, and based on the project, username and password would be picked from the saved list.

This fails of course once we have different view on users, e.g. if we allow more users per project... So I guess we need to wait until we figure out our concept for users...